soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -20,9 +20,12 @@
 #include "rsi_ble_common_config.h"
 #endif
 
+#define AP_MAX_NUM_STA 4
+
 LOG_MODULE_REGISTER(siwx91x_nwp);
 
-int siwg91x_get_nwp_config(int wifi_oper_mode, sl_wifi_device_configuration_t *get_config)
+int siwx91x_get_nwp_config(uint8_t wifi_oper_mode, bool hidden_ssid, uint8_t max_num_sta,
+			   sl_wifi_device_configuration_t *get_config)
```

Comment:
Usually, the returned arguments are placed first in the list of arguments. Rationale, it mimics the usual code:
```
  memcpy(destination, source, ..);
  destination = source;
```